connector/src/main/java/org/geysermc/connector/utils/CooldownUtils.java

```diff
@@ -26,22 +26,15 @@
 package org.geysermc.connector.utils;
 
 import com.nukkitx.protocol.bedrock.packet.SetTitlePacket;
-import org.geysermc.connector.GeyserConnector;
 import org.geysermc.connector.network.session.GeyserSession;
 
-import java.util.concurrent.TimeUnit;
-
 /**
  * Manages the sending of a cooldown indicator to the Bedrock player as there is no cooldown indicator in Bedrock.
  * Much of the work here is from the wonderful folks from ViaRewind: https://github.com/ViaVersion/ViaRewind
  */
 public class CooldownUtils {
 
-    private final static boolean SHOW_COOLDOWN;
-
-    static {
-        SHOW_COOLDOWN = GeyserConnector.getInstance().getConfig().isShowCooldown();
-    }
+    private static boolean SHOW_COOLDOWN;
```

Comment:
Should be camelCase and you can use the getter and setter annotations here.
However I think that you should just check if it's enabled instead of having a getter and a setter in an utils class.